easybuild/easyconfigs/m/MOABS/MOABS-1.3.2-foss-2019b.eb

```diff
@@ -0,0 +1,41 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'MOABS'
+version = '1.3.2'
```

Comment:
@Darkless012 There's a more recent version available via https://github.com/sunnyisgalaxy/moabs (which is also used in Conda), fixed in https://github.com/Darkless012/easybuild-easyconfigs/pull/2